static/js/compose_actions.js

```diff
@@ -118,11 +119,15 @@ function clear_box() {
 }
 
 export function autosize_message_content() {
-    autosize($("#compose-textarea"), {
-        callback() {
-            maybe_scroll_up_selected_message();
-        },
-    });
+    if (compose_height.is_full_size()) {
+        maybe_scroll_up_selected_message();
+    } else {
+        autosize($("#compose-textarea"), {
+            callback() {
+                maybe_scroll_up_selected_message();
+            },
+        });
+    }
```

Comment:
What's the thinking around this function?  It's only called once, when opening compose; is it impossible for compose to be full-size at the moment it's opened?  

Also, do we need to do any scrolling when doing fullsize compose?  This generally needs a comment explaining the reasoning.

Reply:
Yes, it is impossible. The user need to explicitly toggle the full screen mode and only for a particular message.

Well I thought so scrolling may be required, but it isn't required actually.

I actually just added this condition here as we destroy `autosize` when full screen is toggled. So, as a preventive measure if someones calls this function when `autosize` is not declared, it does not give error.

So, I feel this should be done this way. Not adding the change right now, see if this seems good?
```js
if (!compose_height.is_full_size()) {
        autosize($("#compose-textarea"), {
            callback() {
                maybe_scroll_up_selected_message();
            },
        });
    }
```

Reply:
Yeah, that's what I'd had in mind; I'll change that and commit.